_posts/2017-01-18-ruby-meets-rust.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Ruby meets Rust"
+date: 2017-01-18 19:00:00 MEST
+categories: meetup cologne
+links:
+    "Meetup.com": "https://www.meetup.com/RustCologne/events/235877954/"
+    "Cologne.rb": "http://www.colognerb.de/events/januar-meetup-2017"
+    "Planning discussion on Github": "https://github.com/Rustaceans/rust-cologne/issues/19"
+location: coworking-cologne
+talks:
+- title: "Using Rust to make Ruby faster"
+  speaker:
+    name: "Jan-Erik"
+    github: badboy
+    twitter: badboy_
+---
+We are pleased to announce another cooperative meetup, this time with the folks of Cologne.rb, the local Ruby user group.
+So join us on the _third_ Wednesday in January 2017 at Railslove (Coworking Cologne) for an evening of everything that starts with "Ru"!
+
+PS: This meetup will likely be held in German, we will however reevaluate this at the beginning of the evening and may switch to English if needed.
```

Comment:
I suggest
`The language of this meetup is yet to be decided and will be announced soon (either German or English).`